README.md

```diff
@@ -1,2 +1,11 @@
 # nestor-api
-Nestor API
+
+Nestor-api is an API used to manage deployment of applications Kubernetes. I can be used with [nestor-cli](https://github.com/ChauffeurPrive/nestor-cli), a CLI tool consuming nestor-api.
+
+## Installing
+
+Dependencies are managed by [Pipenv](https://github.com/pypa/pipenv).
```

Comment:
We might also want to put at least the Python version required

Reply:
Going even further : if we want to support multiple python version we could have the CI check on each version.
https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#starting-with-the-python-workflow-template
(Since the setup says "at least 3.6")